docs/03-extensions/_category_.yml

```diff
@@ -0,0 +1,2 @@
+label: Extensions
+collapsible: true
```

Comment:
What about splitting extensions by type ? Like :
* Extensions
  * Theming
  * Connectors
  * Payment

Reply:
I think that would be too much nesting again, although it technically is correct.

Reply:
The inital purpose was to keep documentation of each extension (in the monorepos sense) separated, and not to mix everything like we did in v2's documentation.

Reply:
LGTM Then :slightly_smiling_face: